src/site/twitch.tv/modules/player/PlayerModule.vue

```diff
@@ -1,32 +1,67 @@
 <template>
 	<template v-for="inst of player.instances" :key="inst.identifier">
-		<PlayerController :inst="inst" />
+		<PlayerController :inst="inst" :media-player="mediaPlayer" />
+	</template>
+
+	<template v-for="(inst, i) of streamInfo.instances" :key="inst.identifier">
+		<PlayerStreamInfo
+			v-if="shouldShowVideoStats"
+			:inst="inst"
+			:advanced-controls="playerAdvancedOptionsComponent.instances[i]"
+			:media-player="mediaPlayer"
+		/>
 	</template>
 </template>
 
 <script setup lang="ts">
 import { useComponentHook } from "@/common/ReactHooks";
 import { declareModule } from "@/composable/useModule";
-import { declareConfig } from "@/composable/useSettings";
+import { declareConfig, useConfig } from "@/composable/useSettings";
 import PlayerController from "./PlayerController.vue";
+import { ref } from "vue";
+import PlayerStreamInfo from "./PlayerStreamInfo.vue";
 
 declareModule<"TWITCH">("player", {
 	name: "Player",
 	depends_on: [],
 });
 
+const mediaPlayer = ref<Twitch.MediaPlayerInstance>();
+
+const shouldShowVideoStats = useConfig<boolean>("player.video_stats");
+
 const player = useComponentHook<Twitch.VideoPlayerComponent>(
 	{
+		parentSelector: ".persistent-player",
 		predicate: (n) => n.props && n.props.containerRef && n.props.mediaPlayerInstance,
 	},
 	{
 		hooks: {
 			render(inst, cur) {
+				mediaPlayer.value = inst.component.props.mediaPlayerInstance;
+
 				return cur;
 			},
 		},
 	},
 );
+
+// hook for the media player instance advanced controls
+const playerAdvancedOptionsComponent = useComponentHook<Twitch.MediaPlayerAdvancedControls>({
+	parentSelector: ".persistent-player",
+	predicate: (el) => el.props && el.setStatsOverlay,
+});
+
+// hook to render video stats in the channel page view
+const streamInfo = useComponentHook<Twitch.StreamInfo>(
+	{
+		parentSelector: "#live-channel-stream-information",
+		predicate: (el) => el.props && el.props.liveSince,
```

Comment:
Would be nice if these predicates had another prop for specificity, not strictly necessary though.